new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -751,4 +755,58 @@ public static void scaleAndVerifyCluster(String clusterName,
       }
     }
   }
+
+  /**
+   * Get the PodCreationTimestamp of a pod in a namespace.
+   * 
+   * @param namespace Kubernetes namespace that the domain is hosted
+   * @param podName name of the pod 
+   * @return PodCreationTimestamp of the pod
+   */
+  public static String getPodCreationTime(String namespace, String podName) {
+    String podCreationTime =
+        assertDoesNotThrow(() -> getPodCreationTimestamp(namespace, "", podName),
+            String.format("Couldn't get PodCreationTimestamp for pod %s", podName));
+    assertNotNull(podCreationTime, "Got null PodCreationTimestamp");
+    logger.info("PodCreationTimestamp for pod ${0} in namespace ${1} is {2}",
+        namespace,
+        podName,
+        podCreationTime);
+    return podCreationTime;
+  }
+
+  /**
```

Comment:
Please add this method in TestActions.java class.

Reply:
We have already a static method TestActions.getPodCreationTimestamp,  the current convenient utility method uses the same.  By using the utility method we are hiding details of the assertion to be duplicated everywhere.  Some test uses the TestActions.getPodCreationTimestamp directly.

Reply:
Agree for getPodCreationTime. My comment is for patchDomainResourceWithNewRestartVersion method.

Reply:
patchDomainResourceWithNewRestartVersion() is a one-off utility used in MII tests only not a commonly used action.

Reply:
yes, it is still an action right. This is similar to patchDomainCustomResource action method. We need to identify the actions over utility methods, otherwise we will have all methods added as utility methods like in old JUnit 5 TestUtils.java class, it will grow big, that's my only concern.

Reply:
moved the patchDomainResourceWithNewRestartVersion  into TestAction